### PR TITLE
feat: add logo to README

### PR DESCRIPTION
**Pull request by reecejohnson**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<p>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/logo.svg">
+    <img alt="Lens SDK logo" src="./assets/logo/green-with-text.svg" width="auto" height="150">
```
Comment (desfero) [inline README.md:4]: seems the variant for the light theme is missing.

**Review by desfero — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+<p>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/logo.svg">
```
Comment (cesarenaldi) [inline README.md:3]: kinda make sense, just the first time I see this usage!

File: README.md (hunk 1 above)
Comment (reecejohnson) [reply README.md]: thanks, added a version with a bg

File: README.md (hunk 2 above)
Comment (reecejohnson) [reply README.md]: opted to use a bg instead 
image

**Review by krzysu — APPROVED** [review]

**Review by desfero — APPROVED** [review]

**Pull request merged**